src/ray/object_manager/object_manager.cc

```diff
@@ -93,7 +116,10 @@ void ObjectManager::NotifyDirectoryObjectDeleted(const ObjectID &object_id) {
   auto it = local_objects_.find(object_id);
   RAY_CHECK(it != local_objects_.end());
   local_objects_.erase(it);
-  ray::Status status = object_directory_->ReportObjectRemoved(object_id, client_id_);
+  if (local_inlined_objects_.find(object_id) == local_inlined_objects_.end()) {
```

Comment:
Since this set doesn't include all objects that are actually inlined, I wonder if it makes sense to check the same condition as in `HandleObjectAdded` (`(object_info.data_size <= RayConfig::instance().inline_object_max_size_bytes())`), instead of checking for membership in this set. What do you think?

Reply:
Ok, I'll do the change, as it would result in faster code and it's quite clear what we are testing for.